### PR TITLE
[DOC UPDATE] Translate Tutorial to Chinese 9/15

### PR DESCRIPTION
**Pull request by kingswanwho**
# [DOC UPDATE] Translate Tutorial to Chinese 9/15

## Description

Translate a tutorial to Chinese:

080-orchestrating-queries-with-airflow.md

Overall 9 tutorials have been translated, 6 tutorials to go.

## Documentation

Please refer to Chinese tutorial.

## Testing

This is a doc update, no test here.

### CODE REVIEW

File: _docs/zh/tutorials/080-orchestrating-queries-with-airflow.md [hunk 1]
```diff
@@ -278,38 +267,38 @@ with DAG(
...
-- Try adapting the DAG here to work with other data sources.  If you have databases, files and web services in your own environment those will be natural choices, otherwise you can look around online for more public datasets and APIs.
-- Instead of replacing the target dataset, try adding new partitions to an existing dataset by aiming CTAS at date-labelled subdirectories.
-- Keep an eye out for data crunching steps in existing workflows, including those which are not strictly ETL pipelines, where Drill could shoulder some of the load.
+- [了解 Airflow scheduling](https://airflow.apache.org/docs/apache-airflow/1.10.1/scheduler.html) 后台运行 scheduler 使任务自动运行。
+- 调整 DAG 使用其他数据源。如果你在自己的环境中拥有数据库、文件和 Web 服务，那么这些数据源是很好的选择。你也可以在线查看更多公共数据集和 API。
+- 尝试通过将 CTAS 指向带有日期标记的子目录来向现有数据集添加新分区。
+- 留意现有流程中的数据处理步骤，包括那些不是严格意义上 ETL 的步骤，应该交由 Drill 来处理。
 
-Thanks for joining us for this tutorial and happy Drilling!
+感谢完成本教程，希望你喜欢上 Drill。
```
Comment (luocooong) [inline _docs/zh/tutorials/080-orchestrating-queries-with-airflow.md:303]: 希望你已经喜欢上 Apache Drill！

**Review by luocooong — CHANGES_REQUESTED** [review]
@kingswanwho Thanks for the contributions. Please note the comments.

**Comment by kingswanwho** [comment]
All requests resolved

**Review by luocooong — APPROVED** [review]
LGTM. +1

**Pull request merged**